src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -1197,4 +1205,26 @@ public String toString() {
     public LibraryTabContainer getLibraryTabContainer() {
         return tabContainer;
     }
+
+    /**
+     * Takes an entry and increments the number of times it has been viewed by 1.
+     */
+    private void incrementViewCount(BibEntry entry) {
+        MVStore mvStore = PopularityGroup.getMVStore();
+        synchronized (mvStore) {
```

Comment:
if the maps are opened after load, there is no need for `synchronized` (see below)